### PR TITLE
[zh] Translate 2021-04-16-volume-health-monitoring-alpha.md into Chinese

### PR DESCRIPTION
**Pull request by twilight0620**
Fixes: https://github.com/kubernetes/website/issues/35650

2021-04-16-volume-health-monitoring-alpha.md does not have chinese version: https://kubernetes.io/blog/2021/04/16/volume-health-monitoring-alpha-update/

so need to translate it into Chinese

### CODE REVIEW

File: content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+<!--
+The Kubernetes components that monitor the volumes and report events with volume health information include the following:
+-->
+监控卷和使用卷健康信息报送事件的 Kubernetes 组件包括：
+
+<!--
+* Kubelet, in addition to gathering the existing volume stats will watch the volume health of the PVCs on that node. If a PVC has an abnormal health condition, an event will be reported on the pod object using the PVC. If multiple pods are using the same PVC, events will be reported on all pods using that PVC.
+-->
+* Kubelet 除了收集现有的卷统计信息外，还将观察该节点上 PVC 的卷健康状况。
+  如果 PVC 的健康状况异常，则会在使用 PVC 的 pod 对象上报送事件。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md:65]:
```suggestion
  如果 PVC 的健康状况异常，则会在使用 PVC 的 Pod 对象上报送事件。
```

File: content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+<!--
+To use the Volume Health feature, first make sure the CSI driver you are using supports this feature. Refer to this [CSI drivers doc](https://kubernetes-csi.github.io/docs/drivers.html) to find out which CSI drivers support this feature.
+-->
+要使用卷健康功能，首先确保您使用的 CSI 驱动程序支持此功能。
+请参阅此[CSI 驱动程序文档](https://kubernetes-csi.github.io/docs/drivers.html) 以了解哪些 CSI 驱动程序支持此功能。
+
+<!--
+To enable Volume Health Monitoring from the node side, the alpha feature gate `CSIVolumeHealth` needs to be enabled.
+-->
+要从节点侧启用卷健康监控，需要启用 alpha 特性门 `CSIVolumeHealth `。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md:100]:
```suggestion
要从节点侧启用卷健康监控，需要启用 alpha 特性门 `CSIVolumeHealth`。
```

File: content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+<!--
+For details, see the [CSI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md) and the [Kubernetes-CSI Driver Developer Guide](https://kubernetes-csi.github.io/docs/volume-health-monitor.html).
+-->
+有关详细信息，请参阅 [CSI 规约](https://github.com/container-storage-interface/spec/blob/master/spec.md) 
+和 [Kubernetes-CSI 驱动开发者指南](https://kubernetes-csi.github.io/docs/volume-health-monitor.html)。
+
+<!--
+There is a sample implementation for volume health in [CSI host path driver](https://github.com/kubernetes-csi/csi-driver-host-path).
+-->
+[CSI 主机路径驱动程序](https://github.com/kubernetes-csi/csi-driver-host-path) 中有一个卷健康的示例实现。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md:142]:
```suggestion
[CSI 主机路径驱动程序](https://github.com/kubernetes-csi/csi-driver-host-path)中有一个卷健康的示例实现。
```

File: content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+
+<!--
+## How can I learn more?
+-->
+## 如何了解更多？
+
+<!--
+To learn the design details for Volume Health Monitoring, read the [Volume Health Monitor](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1432-volume-health-monitor) enhancement proposal.
+-->
+要了解卷健康监控的设计细节，请阅读 [卷健康监控](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1432-volume-health-monitor) 增强提案。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md:228]:
```suggestion
要了解卷健康监控的设计细节，请阅读[卷健康监控](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1432-volume-health-monitor)增强提案。
```

File: content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+
+<!--
+The Volume Health Monitor controller source code is at [https://github.com/kubernetes-csi/external-health-monitor](https://github.com/kubernetes-csi/external-health-monitor).
+-->
+卷健康检测控制器源代码位于：[https://github.com/kubernetes-csi/external-health-monitor](https://github.com/kubernetes-csi/external-health-monitor).
+
+<!--
+There are also more details about volume health checks in the [Container Storage Interface Documentation](https://kubernetes-csi.github.io/docs/).
+-->
+[容器存储接口文档](https://kubernetes-csi.github.io/docs/) 中还有关于卷健康检查的更多详细信息。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md:238]:
```suggestion
[容器存储接口文档](https://kubernetes-csi.github.io/docs/)中还有关于卷健康检查的更多详细信息。
```

File: content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md [hunk 6]
```diff
@@ -0,0 +1,268 @@
...
+
+<!--
+## How do I get involved?
+-->
+## 如何参与？
+
+<!--
+The [Kubernetes Slack channel #csi](https://kubernetes.slack.com/messages/csi) and any of the [standard SIG Storage communication channels](https://github.com/kubernetes/community/blob/master/sig-storage/README.md#contact) are great mediums to reach out to the SIG Storage and the CSI team.
+-->
+[Kubernetes Slack 频道 #csi](https://kubernetes.slack.com/messages/csi) 和任何 [标准 SIG 存储通信频道](https://github.com/kubernetes/community/blob/master/sig-storage/README.md#contact) 都是联系 SIG Storage 和 CSI 团队的绝佳媒介。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md:248]:
```suggestion
[Kubernetes Slack 频道 #csi](https://kubernetes.slack.com/messages/csi) 和任何[标准 SIG 存储通信频道](https://github.com/kubernetes/community/blob/master/sig-storage/README.md#contact)都是联系 SIG Storage 和 CSI 团队的绝佳媒介。
```

File: content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md [hunk 7]
```diff
@@ -0,0 +1,268 @@
...
+
+<!--
+Depending on feedback and adoption, the Kubernetes team plans to push the CSI volume health implementation to beta in either 1.22 or 1.23.
+-->
+根据反馈和采纳情况，Kubernetes 团队计划在 1.22 或 1.23 中将 CSI 卷健康实施推向 beta。
+
+<!--
+We are also exploring how to use volume health information for programmatic detection and automatic reconcile in Kubernetes.
+-->
+我们还在探索如何在 Kubernetes 中使用卷健康信息进行编程检测和自动协调
```
Comment (Michelle951) [inline content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md:218]: 缺了句号

**Comment by twilight0620** [comment]
I have modified these comments.Thanks for reviewing.

File: content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md [hunk 8]
```diff
@@ -0,0 +1,269 @@
...
+-->
+卷健康监控包括两个部分：
+* 外部卷健康监控控制器从控制器端监控卷健康。
+* Kubelet 从节点端监控卷的健康状况。
+
+<!--
+For details, see the [CSI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md) and the [Kubernetes-CSI Driver Developer Guide](https://kubernetes-csi.github.io/docs/volume-health-monitor.html).
+-->
+有关详细信息，请参阅[CSI 规约](https://github.com/container-storage-interface/spec/blob/master/spec.md)
+和[Kubernetes-CSI 驱动开发者指南](https://kubernetes-csi.github.io/docs/volume-health-monitor.html)。
```
Comment (ydFu) [inline content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md:137]:
```suggestion
有关详细信息，请参阅 [CSI 规约](https://github.com/container-storage-interface/spec/blob/master/spec.md)
和 [Kubernetes-CSI 驱动开发者指南](https://kubernetes-csi.github.io/docs/volume-health-monitor.html)。
```

File: content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md [hunk 9]
```diff
@@ -0,0 +1,269 @@
...
+
+<!--
+To learn the design details for Volume Health Monitoring, read the [Volume Health Monitor](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1432-volume-health-monitor) enhancement proposal.
+-->
+要了解卷健康监控的设计细节，请阅读[卷健康监控](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1432-volume-health-monitor)增强提案。
+
+<!--
+The Volume Health Monitor controller source code is at [https://github.com/kubernetes-csi/external-health-monitor](https://github.com/kubernetes-csi/external-health-monitor).
+-->
+卷健康检测控制器源代码位于：[https://github.com/kubernetes-csi/external-health-monitor](https://github.com/kubernetes-csi/external-health-monitor).
```
Comment (ydFu) [inline content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md:233]:
```suggestion
卷健康检测控制器源代码位于：
[https://github.com/kubernetes-csi/external-health-monitor](https://github.com/kubernetes-csi/external-health-monitor)。
```

File: content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md [hunk 10]
```diff
@@ -0,0 +1,269 @@
...
+<!--
+## How do I get involved?
+-->
+## 如何参与？
+
+<!--
+The [Kubernetes Slack channel #csi](https://kubernetes.slack.com/messages/csi) and any of the [standard SIG Storage communication channels](https://github.com/kubernetes/community/blob/master/sig-storage/README.md#contact) are great mediums to reach out to the SIG Storage and the CSI team.
+-->
+[Kubernetes Slack 频道 #csi](https://kubernetes.slack.com/messages/csi)
+和任何[标准 SIG 存储通信频道](https://github.com/kubernetes/community/blob/master/sig-storage/README.md#contact)都是联系 SIG Storage 和 CSI 团队的绝佳媒介。
```
Comment (ydFu) [inline content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md:249]:
```suggestion
和任何[标准 SIG 存储通信频道](https://github.com/kubernetes/community/blob/master/sig-storage/README.md#contact)都是联系 SIG Storage 和 CSI 团队的绝佳媒介。
```
Why does `SIG Storage` have two translation(`SIG 存储` and `SIG Storage`)?

File: content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md [hunk 11]
```diff
@@ -0,0 +1,270 @@
...
+Michelle Au ([msau42](https://github.com/msau42))、
+David Eads ([deads2k](https://github.com/deads2k))、
+Elana Hashman ([ehashman](https://github.com/ehashman))、
+Seth Jennings ([sjenning](https://github.com/sjenning)) 和 Jiawei Wang ([Jiawei0227](https://github.com/Jiawei0227))
+
+<!--
+Those interested in getting involved with the design and development of CSI or any part of the Kubernetes Storage system, join the [Kubernetes Storage Special Interest Group](https://github.com/kubernetes/community/tree/master/sig-storage) (SIG). We’re rapidly growing and always welcome new contributors.
+-->
+那些有兴趣参与 CSI 或 Kubernetes 存储系统任何部分的设计和开发的人，
+请加入[Kubernetes Storage 特殊兴趣小组](https://github.com/kubernetes/community/tree/master/sig-storage)（SIG）。
```
Comment (ydFu) [inline content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md:269]:
```suggestion
请加入 [Kubernetes Storage 特殊兴趣小组](https://github.com/kubernetes/community/tree/master/sig-storage)（SIG）。
```

File: content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md [hunk 12]
```diff
@@ -0,0 +1,270 @@
...
+<!--
+## How do I use Volume Health on my Kubernetes Cluster?
+-->
+## 如何在 Kubernetes 集群上使用卷健康？
+
+<!--
+To use the Volume Health feature, first make sure the CSI driver you are using supports this feature. Refer to this [CSI drivers doc](https://kubernetes-csi.github.io/docs/drivers.html) to find out which CSI drivers support this feature.
+-->
+要使用卷健康功能，首先确保你使用的 CSI 驱动程序支持此功能。
+请参阅此[CSI 驱动程序文档](https://kubernetes-csi.github.io/docs/drivers.html)以了解哪些 CSI 驱动程序支持此功能。
```
Comment (ydFu) [inline content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md:95]:
```suggestion
请参阅此 [CSI 驱动程序文档](https://kubernetes-csi.github.io/docs/drivers.html)以了解哪些 CSI 驱动程序支持此功能。
```

File: content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md [hunk 13]
```diff
@@ -0,0 +1,270 @@
...
+layout: blog
+title: "Volume Health Monitoring Alpha Update"
+date: 2021-04-16
+slug: volume-health-monitoring-alpha-update
+-->
+
+<!--
+**Author:** Xing Yang (VMware)
+-->
+**作者:** Xing Yang (VMware)
```
Comment (ydFu) [inline content/zh-cn/blog/_posts/2021-04-16-volume-health-monitoring-alpha.md:17]:
```suggestion
**作者：** Xing Yang (VMware)
```

**Review by ydFu — COMMENTED** [review]
/lgtm
thanks

**Pull request merged**